plugins/filterFactory_test.go

```diff
@@ -135,50 +136,71 @@ func TestMakeVolumeFilterConfig(t *testing.T) {
 		wantError   error
 		wantConfig  *VolumeFilterConfig
 	}{
+		{
+			configInput: "volume/daily/buy/base/3500.0/exact",
+			wantConfig: &VolumeFilterConfig{
+				BaseAssetCapInBaseUnits:  pointy.Float64(3500.0),
+				BaseAssetCapInQuoteUnits: nil,
+				mode:                     volumeFilterModeExact,
+				action:                   queries.DailyVolumeActionBuy,
+				additionalMarketIDs:      nil,
+				optionalAccountIDs:       nil,
+			},
+		}, {
+			configInput: "volume/daily/buy/quote/4000.0/exact",
+			wantConfig: &VolumeFilterConfig{
+				BaseAssetCapInBaseUnits:  nil,
+				BaseAssetCapInQuoteUnits: pointy.Float64(4000.0),
+				mode:                     volumeFilterModeExact,
+				action:                   queries.DailyVolumeActionBuy,
+				additionalMarketIDs:      nil,
+				optionalAccountIDs:       nil,
+			},
+		},
 		{
 			configInput: "volume/daily/sell/base/3500.0/exact",
 			wantConfig: &VolumeFilterConfig{
-				SellBaseAssetCapInBaseUnits:  pointy.Float64(3500.0),
-				SellBaseAssetCapInQuoteUnits: nil,
-				mode:                         volumeFilterModeExact,
-				additionalMarketIDs:          nil,
-				optionalAccountIDs:           nil,
+				BaseAssetCapInBaseUnits:  pointy.Float64(3500.0),
+				BaseAssetCapInQuoteUnits: nil,
```

Comment:
Sounds good. Since this is updating an old test, I figured that like our more recent tests, we'd want to vary both mode and action. To do this more efficiently, I changed the `configInput` to a format string and injected both mode and action. If the previous way would be preferred, let me know and I'm happy to change it back.

Reply:
the previous way is a little better because it's more readable, a direct mapping from input to output.